webpack/assets/javascripts/react_app/components/Bookmarks/components/BookmarkForm/__tests__/__snapshots__/BookmarkForm.test.js.snap

```diff
@@ -67,7 +67,6 @@ exports[`BookmarkForm should render bookmarks form with initial values 1`] = `
           "transforms": Array [
             [Function],
           ],
-          "type": "string",
```

Comment:
Fixed failed snapshot - not related to this PR

Reply:
the fix was just merged, please rebase on develop.

Reply:
rebased, and I still see it, also locally

Reply:
@MariaAga can you take a look please?

Reply:
Is the vendor version is `4.14.6`? might need to remove `node_modules` and `npm i` to update

Reply:
thanks, before today's last forced push I removed node_modules,
but there seem last run was just between the new foreman-vendor release and Maria fix,
now it works locally, lets wait for tests here.